src/org/usfirst/frc/team125/robot/commands/RunIntakeForwardCMD.java

```diff
@@ -9,40 +9,38 @@
 
 import edu.wpi.first.wpilibj.command.Command;
 import org.usfirst.frc.team125.robot.Robot;
+import org.usfirst.frc.team125.robot.subsystems.Intake;
 
 /**
  * An example command.  You can replace me with your own command.
  */
-public class ExampleCommand extends Command {
-	public ExampleCommand() {
-		// Use requires() here to declare subsystem dependencies
-		requires(Robot.kExampleSubsystem);
+public class RunIntakeForwardCMD extends Command {
+
+	public RunIntakeForwardCMD() {
+		requires(Robot.intake);
 	}
 
-	// Called just before this Command runs the first time
 	@Override
 	protected void initialize() {
 	}
 
-	// Called repeatedly when this Command is scheduled to run
 	@Override
 	protected void execute() {
+		Robot.intake.runIntake(.75);
```

Comment:
since we need to play around with the peak outputs, if this number is 1.0, it might be easier to find what the peak outputs will need to be; just a suggestion.